domain/model/mesh_results.py

```diff
@@ -105,21 +105,27 @@ def connection(self, from_agent, to_agent: AgentID) -> MeshColumn:
 
 
 class MeshResults:
-    """Internal representation of Mesh Test results; independent of source data structure like http or grpc synthetics client"""
+    """
+    Internal representation of Mesh Test results; independent of source data structure like http
+    or grpc synthetics client
+    """
 
     def __init__(
         self,
-        utc_timestamp: Optional[datetime] = None,
-        rows: List[MeshRow] = [],
+        utc_timestamp: datetime,
+        rows: Optional[List[MeshRow]] = None,
         agents: Agents = Agents(),
     ) -> None:
         if utc_timestamp is None:
```

Comment:
This should be removed as utc_timestamp can no longer be None

Reply:
Well it can, because caller can pass `None` to the constructor. Mypy would though complain. I'll remove the test.